src/main/java/se/sundsvall/snailmail/api/SnailMailResource.java

```diff
@@ -70,4 +76,12 @@ ResponseEntity<Void> sendBatch(
 		return ok().build();
 	}
 
+	// Determine the value of the "sentBy" header, if present use it, otherwise try to get the value from the
+	// x-issuer-header
+	private String resolveSentBy(final String issuer) {
+		return Optional.ofNullable(Identifier.get())
+			.map(Identifier::getValue)
+			.filter(StringUtils::isNotBlank)
```

Comment:
Filter on `Identifier.getType().equals(Type.AD_ACCOUNT)` 

or can issuer be a partyId (or something else) as well?